Audacia.Log.AspNetCore/Audacia.Log.AspNetCore.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <Copyright>Copyright © Audacia 2019</Copyright>
```

Comment:
Should we update this from `Copyright © Audacia 2019` to `Copyright © Audacia $([System.DateTime]::Now.Year)`

Reply:
@Audacia-RhysSmith This has been actioned